src/Illuminate/Database/Console/DatabaseInspectionCommand.php

```diff
@@ -19,13 +19,15 @@ abstract class DatabaseInspectionCommand extends Command
      * @param  \Illuminate\Database\ConnectionInterface  $connection
      * @param  string  $database
      * @return string
+     *
+     * @deprecated Use $connection->getName() instead.
```

Comment:
We can deprecate this, these are not the same, actually `$connection->getName()` refers to an array-key defined on `config('database.connections')` array, e.g `pgsql`, `mysql`, `sqlsrv`.

We still need pretty names!